pytket/extensions/iqm/backends/iqm.py

```diff
@@ -104,10 +99,7 @@ def __init__(
         :py:meth:`pytket.extensions.iqm.set_iqm_config`.
 
         :param url: base URL for requests
-        :param settings: path of JSON file containing device settings
-        :param arch: list of couplings between the qubits defined in the device settings
-            (default: [("QB1", "QB3"), ("QB2", "QB3"), ("QB4", "QB3"), ("QB5", "QB3")],
-            i.e. a 5-qubit star topology centred on "QB3")
+        :param arch: list of couplings between the qubits defined
```

Comment:
If (as stated in the README) this is normally not required, we should make that clear here.

Reply:
Thank you, done